app/assets/stylesheets/visualizations.css.scss

```diff
@@ -406,56 +303,162 @@ body[data-page-name='visualizations/show'] .visualizations-controller,
+  .vis-canvas {
+    height: inherit;
+    width: inherit;
+    overflow: hidden; // because something inside vis-canvas
+                      // is overflowing on every non-map vis
   }
 
-  // just used for the border around the vis window
-  #viscontainer {
+  #vis-wrapper {
     width: 100%;
-    height: 98%;
-    font-size:14px;
-    min-height:500px;
-    background-color: #FFF; // firefox - control menu turns black otherwise
-    border: $border;
-  }
+    height: 100%;
 
-  .vis_controls {
-    float: right;
-    width:100%;
-  }
+    // floats the actual vis to the right
+    #vis-container {
+      position: absolute;
+      right: 0;
+      width: 100%;
+      height: 100%;
 
-  #vis-disabled {
-    overflow: hidden;
-  }
+      .tab-content {
+        width: 100%;
+      }
 
-  // the bar at the top of the vis window
-  #visTabList {
-    background-image: $light-bkg;
-    border-bottom: $border; // puts a border only on the bottom of the tab list, #viscontainer does top and sides
-    padding: 0; // removes padding that shouldn't be there to begin with
-    margin: 0;
-  }
+      #vis-disabled {
+        overflow: hidden;
+      }
 
-  // an individual tab
-  .vis_tab {
-    @include unselectable;
-    display: inline-block;
-    font-size:1.2em;
-    border-right: $border; // borders are only on the right - tab list and vis window cover the other sides
-
-    .ui-tabs-anchor {
-      display: block; // so vertical padding works
-      padding: 6px 10px; // so the entire box is clickable
-      color: #222;
-      outline: none;
+      // the bar at the top of the vis window
+      #vis-tab-list {
+        padding: 0; // removes padding that shouldn't be there to begin with
+        margin: 0;
+        background-color: $vis-bkg-light;
+      }
+
+      #vis-title-bar {
+        position: relative;
+        width: 100%;
+        height: 50px;
+        background-color: $vis-bkg-light;
+        font-size: 24px;
+        padding: 10px;
+        white-space: nowrap;
+
+        .edit_visualization, .edit_data_set {
+          display: none;
+        }
+
+        .inline-left {
+          max-width: 100%;
+        }
+
+        #title-and-menu {
+          display: inline-block;
+
+          #title-name {
+            vertical-align: middle;
+          }
+        }
+      }
+    }
+
+    #vis-ctrl-container {
+      position: relative;
+      display: inline-block;
+      float: left;
+      background-color: $vis-bkg-dark;
+      width: 320px;
+      height: 100%;
+
+      // this is where all the div controls get added
+      #vis-ctrls {
+        height: 100%;
+        overflow-x: hidden;
+        overflow-y: auto;
+        position: relative;
+        display: inline-block;
+        float: left;
+
+        // puts spacing between some boostrap buttons
+        .form-control { // this styles bootstrap stuff and IS VERY BAD
+          margin-bottom: 5px;
```

Comment:
@asalani93 You added this originally - I changed the indentation. Any idea if this can be fixed / why it's bad?